src/deposit.cairo

```diff
@@ -325,6 +353,32 @@ mod Deposit {
                     }
                 );
         }
+
+        /// Claims STRK airdrop on ZKlend
+        ///
+        /// # Panics
+        /// `is_position_open` storage variable is set to false('Open position not exists')
+        /// `proof` span is empty
+        /// Claim call failed
+        ///
+        /// # Parameters
+        /// `claim_data`: Claim - contains data about claim operation
+        /// `proof`: Span<felt252> - proof used to validate the claim
+        /// `airdrop_addr`: ContractAddress - address of a contract responsible for claim
+        fn claim_reward(
+            ref self: ContractState,
+            claim_data: Claim,
+            proof: Span<felt252>,
+            airdrop_addr: ContractAddress
+        ) {
+            assert(self.is_position_open.read(), 'Open position not exists');
+            assert(proof.len() != 0, 'Proof Span cannot be empty');
```

Comment:
I appreciate the defensiveness, but this is not necessary; it will fail when calling claim() anyway